lib/msal-common/src/network/INetworkModule.ts

```diff
@@ -14,5 +14,5 @@ export interface INetworkModule {
      * @param requestParams 
      * @param enableCaching 
      */
-    sendRequestAsync(url: string, requestParams: RequestInit, enableCaching?:boolean): Promise<any>
+    sendRequestAsync(url: string, requestParams: RequestInit): Promise<any>;
```

Comment:
Instead of `RequestInit`, implement our own type that is platform/api agnostic.

e.g.
```js
type RequestParams {
   body?: object,
   headers?: object
}
```